library/build.gradle

```diff
@@ -41,14 +41,32 @@ android {
 }
 
 ext {
-    libtorrentVersion = '1.2.1.0'
+    libtorrentVersion = '1.2.5.0'
+}
+
+// Custom task which downloads the appropriate version of JAR files for jlibtorrent
+task downloadDependencies(type: Download) {
```

Comment:
Do they no longer publish the libraries on maven?

Reply:
It doesn't look like it 👎

Feel free to verify that, to make sure I'm not going crazy or that it's not some network / transient issue on my end.

Reply:
It looks like one of the jlibtorrent developers has forked the project some time ago and that version is available via Maven: https://github.com/aldenml/libtorrent4j. Since then there have been no maven releases anymore.

I guess this is a good solution for know, but switching to libtorrent4j may be even better.